airflow/models/dagbag.py

```diff
@@ -408,13 +405,9 @@ def collect_dags(
         self.dagbag_stats = sorted(
             stats, key=lambda x: x.duration, reverse=True)
         for file_stat in self.dagbag_stats:
-            dag_ids = dags_by_name[file_stat.dags]
-            if file_stat.dag_num >= 1:
-                # if we found multiple dags per file, the stat is 'dag_id1 _ dag_id2'
-                dag_names = '_'.join(dag_ids)
-                Stats.timing('dag.loading-duration.{}'.
-                             format(dag_names),
-                             file_stat.duration)
+            Stats.timing('dag.loading-duration.{}'.
+                         format(file_stat.file),
```

Comment:
Question: Is `file_stat.file` just the file name or it is the file path that contains `/`?

Reply:
Good point, I double check and the file name is `/subdag/dagname.py` something. Update the pr.